src/UseCases/ModerateMembershipApplication/ModerateMembershipApplicationResponse.php

```diff
@@ -7,11 +7,10 @@
 use WMDE\Fundraising\MembershipContext\UseCases\SimpleResponse;
 
 class ModerateMembershipApplicationResponse implements SimpleResponse {
-	private int $membershipApplicationId;
+
 	private bool $success;
 
-	private function __construct( int $membershipApplicationId, bool $isSuccess ) {
-		$this->membershipApplicationId = $membershipApplicationId;
+	private function __construct( private readonly int $membershipApplicationId, bool $isSuccess ) {
```

Comment:
```suggestion
	private function __construct( private readonly int $membershipApplicationId, private readonly bool $success ) {
```

Reply:
@Abban 
I talked with Gabriel about this and I think he prefers `$isSuccess`, because it's easier for the getters and setters. @gbirke Would you please present the reasoning again?

Reply:
The internal property is called `success` because the pubic accessor method is called `isSuccess` while creating this class the author's mind was probably in JavaScript-land where it's forbidden to have matching properties and method. TIL in PHP [a class can have properties and methods with the same name](https://www.php.net/manual/en/language.oop5.basic.php#language.oop5.basic.properties-methods). However we might want to discuss this in the team if we really want to use this language feature, to avoid clashes with other languages that disallow that.

The thinking behind naming the constructor variable was this: In a constructor we're often passing in properties, but the ones that are visible to the outside. Therefore I think matching variable names of the public accessor and the "public" constructor parameter name (you can see the name in docs and the IDE) is a better consistency than the constructor parameter being consistent with a private property.

Would you agree, @Abban @moiikana @Sperling-0 ?

And what's your stance on matching names of properties and methods?

Reply:
I was only indicating that you can replace the internal private property (which is named `$success`) with property promotion. You can also change that to `$isSuccess`. IMO it's totally fine to name them the same in this case as it's a private property.